packages/components/src/Banner/Banner.tsx

```diff
@@ -1,14 +1,14 @@
 import Heading, { HeadingElement } from "../Heading";
+import NotificationIcon, {
+  NotificationVariant,
+} from "../common/Notifications/NotificationIcon";
 import Button from "../Button";
-import CloseIcon from "../Icons/Close";
-import NotificationIcon from "../common/Notifications/NotificationIcon";
```

Comment:
This is currently a circular reference b/c `NotificationIcon` imports `BannerColor` 😬  Added a backlog task to add some checking for circular dependencies: https://app.clubhouse.io/czi-edu/story/152408/check-for-circular-dependencies